### PR TITLE
README->Installation: Clarify that the alias is really needed

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

**Comment by nvbn** [comment]
In newer versions when alias isn't configured it works like:
```bash
➜  fuck
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.zshrc and apply changes with source ~/.zshrc or restart your shell.
Or run fuck second time for configuring it automatically.
More details - https://github.com/nvbn/thefuck#manual-installation
```

**Comment by sphvn** [comment]
@nvbn, the current version displaying the message like you posted above is more than enough.

👍 on just closing this in my opinion.

**Comment by matkoniecz** [comment]
Bold is overkill, but changing "should" to "must" in my opinion is a good clarification.

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ pip install --user thefuck
 
 [Or using an OS package manager (OS X, Ubuntu, Arch).](https://github.com/nvbn/thefuck/wiki/Installation)
 
-You should place this command in your `.bash_profile`, `.bashrc`, `.zshrc` or other startup script:
+**Now you must place this command in your `.bash_profile`, `.bashrc`, `.zshrc` or other startup script:**
```
Comment (scorphus) [inline README.md:120]:
```suggestion
You must place this command in your `.bash_profile`, `.bashrc`, `.zshrc` or other startup script:
```

**Review by scorphus — CHANGES_REQUESTED** [review]

**Comment by scorphus** [comment]
Actually, the section of REAME.md related to this PR has changed and is clear enough, IMO. Closing this PR for now.

**Pull request closed without merging**